SpringProject/src/main/java/com/gradproject2019/userConference/service/UserConferenceServiceImpl.java

```diff
@@ -38,11 +45,32 @@ public UserConferenceResponseDto saveInterest(UUID token, UserConferenceRequestD
         }
     }
 
+    @Override
+    public List<ConferenceResponseDto> getConferenceByUserId(UUID token, Long userId) {
+        List<ConferenceResponseDto> conferenceResponseDtos = new ArrayList<>();
```

Comment:
There's no point creating the list if the user isn't authorized, so this could go after the next line,